editor/sidebar/post-visibility/index.js

```diff
@@ -11,6 +11,7 @@ import { withInstanceId } from 'components';
  */
 import { __ } from 'i18n';
 import { Component } from 'element';
+import { ESCAPE } from 'utils/keycodes';
```

Comment:
Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```

Further, we should not import from nested paths. See #941. Instead, `import { keycodes } from '@wordpress/utils';` and destructure from the `keycodes` object:

```js
const { ESCAPE } = keycodes;
```